staging/src/k8s.io/cri-api/README.md

```diff
@@ -17,89 +17,40 @@ to be Kubernetes-centric. We try to avoid it, but there may be logic within a co
-different versions of CRI API.
+Please read about:
 
-For example, on a single node there might be:
+- [CRI API version skew policy](https://kubernetes.dev/docs/code/cri-api-version-skew-policy/)
+- [Kubernetes feature development and container runtimes](https://kubernetes.dev/docs/code/cri-api-dev-policies/)
 
-- _Kubelet_ may call into _Container Runtime_ (e.g. [containerd](https://containerd.io))
-  and _Image Service Proxy_ (e.g. [stargz-snapshotter](https://github.com/containerd/stargz-snapshotter)).
-  _Container Runtime_ may be versioned with the OS Image, _Kubelet_ is installed
-  by system administrator and _Image Service proxy_ is versioned by the third party vendor.
-- _Image Service Proxy_ calls into _Container Runtime_.
-- _CRI tools_ (e.g. [crictl](https://kubernetes.io/docs/tasks/debug/debug-cluster/crictl/))
-  may be installed by end user to troubleshoot, same as a third party daemonsets.
-  All of them are used to call into the _Container Runtime_ to collect container information.
-
-So on a single node it may happen that _Container Runtime_ is serving a newer
-version'd kubelet and older versioned crictl. This is a supported scenario within
-the version skew policy.
-
-### Version Skew Policy for CRI API
-
-CRI API has two versions:
-- Major semantic version (known versions are `v1alpha2` ([removed in 1.26](https://kubernetes.io/blog/2022/12/09/kubernetes-v1-26-release/#cri-v1alpha2-removed)), `v1`).
-- Kubernetes version (for example: `@1.23`). Note, the `cri-api` Golang library is versioned as `0.23` as it doesn't guarantee Go types backward compatibility.
-
-Major semantic version (e.g. `v1`) is used to introduce breaking changes
-and major new features that are incompatible with the current API.
-
-Kubernetes version is used to indicate a specific feature set implemented
-on top of the major semantic version. All changes made without the change
-of a major semantic version API must be backward and forward compatible.
-
-- _Kubelet_ must work with the older _Container Runtime_ if it implements
-  the same semantic version of CRI API (e.g. `v1`) of up to three Kubernetes minor
-  versions back. New features implemented in CRI API must be gracefully degraded.
-  For example, _Kubelet_ of version 1.26 must work with _Container Runtime_
-  implementing `k8s.io/cri-api@v0.23.0`+.
-- _Kubelet_ must work with _Container Runtime_ if it implements
-  the same semantic version of CRI API (e.g. `v1`) of up to
-  three minor versions up. New features implemented in CRI API must not change
-  behavior of old method calls and response values. For example, _Kubelet_ of
-  version 1.22 must work with _Container Runtime_ implementing `k8s.io/cri-api@v0.25.5`.
-
-
-## Versioning
+## Community, discussion, contribution, and support
 
-This library contains go classes generated from the CRI API protocol buffers and gRPC API.
+Learn how to engage with the Kubernetes community on the [community
+page](http://kubernetes.io/community/).
```

Comment:
Could link to https://www.k8s.dev/community/

Reply:
applied